pkg/storage/store.go

```diff
@@ -3752,9 +3752,8 @@ func (s *Store) updateReplicationGauges(ctx context.Context) error {
 			// If this replica is the highest replica ID, it does the counting.
 			if highestIdx != -1 && desc.Replicas[highestIdx].StoreID == s.StoreID() {
 				rangeCount++
-				// If a quorum of replicas are live, consider this range available.
-				if liveReplicas > computeQuorum(len(desc.Replicas)) {
```

Comment:
The bug was just that we were not specifying `>=` here.

Changing the metric itself will be very disruptive and this PR doesn't address the various places in the admin UI that are looking for `availableRanges`.